noisereduction/TrackUtils.cpp

```diff
@@ -1,7 +1,33 @@
 #include "TrackUtils.h"
-#include "loguru.hpp"
+
 #include <memory>
+#include <stdexcept>
+
 #include "MemoryX.h"
+#include "loguru.hpp"
+
+#define WINDOWS (defined(WIN32) || defined(_WIN32) || defined(__WIN32))
```

Comment:
btw if you want to try out something cool, because you also control the build system, you can use the CMakeLists.txt script to add a USE_MMAP define (https://cmake.org/cmake/help/latest/command/target_compile_definitions.html)

You can then just #ifdef on that instead of checking for Windows here :)

Reply:
Thank you, that sounds interesting 👍🏻